DOL.WHD.Section14c.Web/src/modules/components/mainNavigationControl/mainNavigationControlController.js

```diff
@@ -25,13 +25,30 @@ module.exports = function(ngModule) {
     vm.current = $location.$$path.split('/section/')[1];
     vm.collapseMenu = true; //collapse menu by default for small screens
     this.onNavClick = function(event) {
+      function toTitleCase(str) {
+      return str.replace(/\w\S*/g, function (txt) {
+          return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();
+        });
+      }
+
       var id = event.target.dataset.sectionid;
       navService.gotoSection(id);
-    };
+    
+
+    if (id === 'app-info') {
+      document.title = 'Application Info | DOL WHD Section 14(c)';
+    } else if (id === 'work-sites') {
+      document.title = 'Work Sites & Employees | DOL WHD Section 14(c)';
+    } else if (id === 'wioa') {
+      document.title = 'WIOA | DOL WHD Section 14(c)';
+    } else {
+      id = id.replace(/-/g, ' ');
+      document.title = toTitleCase(id) + ' | DOL WHD Section 14(c)';
+    }
 
     this.onKeyPress = e => {
       if (e.which === 13) this.onNavClick(e);
     };
-
+  };
```

Comment:
Can you fix the indention on lines 29 and 38-52?  Just a minor consistency thing.

Reply:
I'd also like to see the tests for this class extended to check that the title is set correctly.  Something like this in `mainNavigationControlController.test.js` should do the trick:

```js
  describe('sets the document title correctly', function() {
    var controller, event;
    beforeEach(function() {
      controller = mainNavigationControlController();
      spyOn(mockNavService, 'gotoSection');
      event = { target: { dataset: { sectionid: '' } } };
    });

    it('sets the title for app-info', function() {
      event.target.dataset.sectionid = 'app-info';
      controller.onNavClick(event);
      expect(document.title).toBe('Application Info | DOL WHD Section 14(c)');
    });

    it('sets the title for work-sites', function() {
      event.target.dataset.sectionid = 'work-sites';
      controller.onNavClick(event);
      expect(document.title).toBe('Work Sites & Employees | DOL WHD Section 14(c)');
    });

    it('sets the title for wioa', function() {
      event.target.dataset.sectionid = 'wioa';
      controller.onNavClick(event);
      expect(document.title).toBe('WIOA | DOL WHD Section 14(c)');
    });

    it('sets the title for unhandled-case', function() {
      event.target.dataset.sectionid = 'unhandled-case';
      controller.onNavClick(event);
      expect(document.title).toBe('Unhandled Case | DOL WHD Section 14(c)');
    });
  });
```